x-pack/plugins/security_solution/public/management/cypress/e2e/automated_response_actions/results.cy.ts

```diff
@@ -61,15 +60,17 @@ describe('Results', { tags: ['@ess', '@serverless'] }, () => {
     () => {
       before(() => {
         login(ROLE.endpoint_response_actions_access);
-        disableExpandableFlyoutAdvancedSettings();
       });
 
       it('see endpoint action', () => {
-        cy.visit(APP_ALERTS_PATH);
+        navigateToAlertsList(`query=(language:kuery,query:'_id: ${alertData?.alerts[0]._id}')`);
```

Comment:
why do we need this change? I mean it's good to narrow down the view, but it should not be necessary in this case right? Or does it open the flyout automatically with this change?

Reply:
it improves the stability of the test